front/app/components/EsriMap/components/EsriMapMeta.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Helmet } from 'react-helmet-async';
+
+const EsriMapMeta = () => {
+  return (
+    <Helmet>
+      <meta name="referrer" content="origin" />
+    </Helmet>
+  );
```

Comment:
This should hopefully do the trick. I was able to toggle on/off sending referrers locally with this method, but I'm not sure how that nginx config will work with this :thinking: 

I'll test it out on staging first to see at least, since the problem exists there as well.